crates/iota-graphql-rpc/src/types/object.rs

```diff
@@ -226,6 +226,14 @@ pub(crate) enum ObjectLookup {
         /// The checkpoint sequence number at which this was viewed at.
         checkpoint_viewed_at: u64,
     },
+
+    /// Variant analogous to VersionAt but for optimistic transactions,
```

Comment:
not sure if it's 100% correct syntactically

```suggestion
    /// Variant analogous to [`VersionAt`](Self::VersionAt) but for optimistic transactions,
```

Reply:
Done in 7bf05db07bfdc1a7f55afa62b4070461c3e95976